openapi/views/openapi_view.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id='openapi_namespace_model_view_tree' model='ir.ui.view'>
+        <field name="name">openapi.namespace.tree</field>
+        <field name="model">openapi.namespace</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="namespace_form_view">
+        <field name="name">openapi.namespace.form</field>
+        <field name="model">openapi.namespace</field>
+        <field name="arch" type="xml">
+            <form string="Namespace">
+              <sheet>
+                <div class="oe_button_box" name="button_box">
+                    <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
+                        <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
+                    </button>
+                </div>
+                <div class="oe_title">
+                    <label for="name" class="oe_edit_only"/>
+                    <h1>
+                        <field name="name" class="oe_inline"/>
+                    </h1>
+                    <label for="description" class="oe_edit_only"/>
+                    <h3>
+                        <field name="description"/>
+                    </h3>
+                    <field name="id" invisible="1"/>
+                    <group>
+                        <field name="log_request"/>
+                        <field name="log_response"/>
+                        <field name="last_log_date" readonly="1"/>
+                        <field name="token"/>
+                        <field name="spec_url" widget="url" readonly="1"/>
+                    </group>
+                </div>
+                <notebook>
+                    <page string="Accessable models">
+                        <field name="access_ids"
```

Comment:
@RafiZz The readonly should be removed from here and from other similar places. Look at an example in standard odoo modules. E.g. the sale module(sale.order and sale.order.line models)